Branch-SDK/src/main/java/io/branch/referral/ServerRequestInitSession.java

```diff
@@ -55,6 +52,9 @@ protected void setPost(JSONObject post) throws JSONException {
         if (!DeviceInfo.isNullOrEmptyOrBlank(appVersion)) {
             post.put(Defines.Jsonkey.AppVersion.getKey(), appVersion);
         }
+        if(prefHelper_.getInitialReferrer() != null && !prefHelper_.getInitialReferrer().equals(PrefHelper.NO_STRING_VALUE)) {
```

Comment:
Another nitpick: instead of `prefHelper_.getInitialReferrer()`, let's do `!TextUtil.isEmpty(prefHelper_.getInitialReferrer())`